includes/login.php

```diff
@@ -315,31 +315,47 @@ function pmpro_login_forms_handler( $show_menu = true, $show_logout_link = true,
 	$message = '';
 	$msgt = 'pmpro_alert';
 	if ( isset( $_GET['action'] ) ) {
+		$username = isset( $_GET['username'] ) ? esc_html( $_GET['username'] ) : '';
```

Comment:
```suggestion
		$username = isset( $_GET['username'] ) ? sanitize_text_field( $_GET['username'] ) : '';
```

Reply:
I used esc_html here as it was outputting to the screen.

Reply:
That above comment was done directly from VSCode, that's cool! :)